scripts/uosc/lib/std.lua

```diff
@@ -146,13 +156,13 @@ function itable_slice(itable, start_pos, end_pos)
 end
 
 ---@generic T
----@param a T[]|nil
----@param b T[]|nil
+---@param ...T[]|nil
 ---@return T[]
-function itable_join(a, b)
+function itable_join(...)
 	local result = {}
-	if a then for _, value in ipairs(a) do result[#result + 1] = value end end
-	if b then for _, value in ipairs(b) do result[#result + 1] = value end end
+	for _, table in ipairs({...}) do
+		if table then for _, value in ipairs(table) do result[#result + 1] = value end end
```

Comment:
Is it actually necessary to check table for nil?

Reply:
As the function's signature suggests, I want it to accept stuff like `itable_join(array, nil, array, ...)` and just ignore nils, but after testing it, the iterator thinks `nil` is an end of the array and stops there. I'll fix it.